### PR TITLE
Use contributing review fixes

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Fixes comments from #46

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_EXERCISM_HOST=http://lvh.me:3000
```
Comment (SleeplessByte) [inline .env.development:1]: I recommend `.gitignore` this file.

File: src/components/TrackNewExercise.tsx [hunk 2]
```diff
@@ -86,7 +97,7 @@ export function TrackNewExercise(props: TrackNewExerciseProps): JSX.Element {
     ].join('\n\n')
     const instructions = `${story}\n\n${tasks}`
 
-    fetch('https://exercism.io/git_api/concept_exercises', {
+    fetch(`${process.env.REACT_APP_EXERCISM_HOST}/git_api/concept_exercises`, {
```
Comment (SleeplessByte) [inline src/components/TrackNewExercise.tsx:100]: You can consider extracting this to a top-level constant, and giving a resolvable error if it's not present:

```
const EXERCISM_HOST: string | undefined = process.env.REACT_APP_EXERCISM_HOST
if (!EXERCISM_HOST) {
  throw new Error('REACT_APP_EXERCISM_HOST environment variable missing. Add it to a .env file to resolve this issue').
}

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -1 +1,7 @@
 /// <reference types="react-scripts" />
+
+declare namespace NodeJS {
+  interface ProcessEnv {
+    readonly REACT_APP_EXERCISM_HOST: string
```
Comment (SleeplessByte) [inline src/react-app-env.d.ts:5]:
```suggestion
    readonly REACT_APP_EXERCISM_HOST: string | undefined
```

File: .env.development (hunk 1 above)
Comment (ErikSchierboom) [reply .env.development]: I've gone a slightly different route after reading through [the docs](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env). I now have a single `.env` file. Locally, I can then have a `.env.development.local` file that overrides this. This works nicely!

File: src/components/TrackNewExercise.tsx (hunk 2 above)
Comment (ErikSchierboom) [reply src/components/TrackNewExercise.tsx:100]: Where should this top-level constant be defined? The URL is used in multiple places

**Pull request merged**